guides/v2.1/cloud/howtos/debug.md

```diff
@@ -231,7 +231,7 @@ To start debugging, use the following commands with the environment URL:
 
         ?XDEBUG_SESSION_STOP=KEY
 
-Please note, `XDEBUG_SESSION_START`passed by `POST` requests are not supported at this time.
+Note, `XDEBUG_SESSION_START`passed by `POST` requests are not supported at this time.
```

Comment:
Note that ....  or use Note formatting?

Reply:
Yeah, I wonder why they did it that way? This topic is full of grievances.